nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceInitializationContext.java

```diff
@@ -87,4 +87,5 @@ public ComponentLog getLogger() {
     public StateManager getStateManager() {
         return stateManager;
     }
+
```

Comment:
nit-pick: this is the only change to this file (line at the end). Can it be removed?